.circleci/config.yml

```diff
@@ -27,6 +35,6 @@ jobs:
       - store_artifacts:
           path: ./coverage
       - save_cache:
```

Comment:
You may want to move the cache up a bit to speed up work in progress where lint/build/test might be failing.